coalib/settings/Setting.py

```diff
@@ -108,7 +125,7 @@ def __init__(self,
         self.key = key
         self.origin = str(origin)
 
-    def __path__(self, origin=None):
+    def __path__(self, origin=None, globbed_origin=False):
```

Comment:
`path(Setting(...), origin="xyz", is_glob=True`
vs
`path(Setting(...), origin="xyz", glob_escape_origin=True`
I think both are good to read, since the path you get in 1. `is_` a `glob` :)
and in second you say it explicitly.
Yeah your choice :+1: :)